src/toggle/util/toggleHooks.ts

```diff
@@ -0,0 +1,12 @@
+import {useContext} from "react";
+
+import {ToggleContext} from "../Toggle";
+
+/**
+ * @returns {Object} Current value of ToggleContext
+ */
+function useToggleContext() {
+  return useContext(ToggleContext);
+}
+
+export {useToggleContext};
```

Comment:
Why is this included here ? 🤔

Reply:
Ops! It's really weird. I guess my local branches are confused.